src/logistration/LoginHelpLinks.jsx

```diff
@@ -22,8 +23,12 @@ const LoginHelpLinks = (props) => {
     setShowLoginHelpValue(!showLoginHelp);
   };
 
+  const handleForgotPasswordLinkClickEvent = () => {
+    sendGAEvent('analytics', 'edx.bi.password-reset_form.toggled', { category: 'user-engagement' });
+  };
+
   const forgotPasswordLink = () => (
-    <a className="field-link" href={RESET_PAGE}>
+    <a className="field-link" href={RESET_PAGE} onClick={(e) => handleForgotPasswordLinkClickEvent(e)}>
```

Comment:
I guess we don't need the click event `e` in `handleForgotPasswordLinkClickEvent` and remove it form the call?
```suggestion
    <a className="field-link" href={RESET_PAGE} onClick={handleForgotPasswordLinkClickEvent}>
```